### PR TITLE
IAM-506: Add Kafka client and blackhole

### PR DESCRIPTION
**Pull request by thoriqsatriya**
* The Kafka client still using the default configuration, more configuration like backoff retry will be added later

### CODE REVIEW

**Comment by thoriqsatriya** [comment]
please help review this @rjjatson @ajiyakin @leowind @vincent6767

**Comment by vincent6767** [comment]
@thoriqsatriya Are we going to have Kafka integration tests also?

**Comment by thoriqsatriya** [comment]
@vincent6767 It's better to have integration test, for now I think just use mock first. The reasons is, we don't sure yet about which Kafka we're going to use. And the other thing is I'm not sure what Travis CI plan that we're using, since Kafka need big resource to run locally.
What do you think?

File: makefile [hunk 1]
```diff
@@ -1,8 +1,25 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ clean:
 	rm coverage.out
 
 test:
-	go test -cover ./...
+	docker-compose -f docker-compose-test.yml up -d
+	sleep 30
```
Comment (vincent6767) [inline makefile]: What's the reason we have this?

Comment (aripnk) [reply makefile]: I think it is waiting for the docker-compose service available

Comment (thoriqsatriya) [reply makefile]: Yeah, that's correct. It's for witing the docker-compose to be ready.

**Pull request merged**